package/build/LibSassBuilder.props

```diff
@@ -14,18 +14,28 @@
     <EnableDefaultSassItems Condition="'$(EnableDefaultSassItems)'==''">true</EnableDefaultSassItems>
     <DefaultSassExcludes>**/bin/**;**/obj/**;**/node_modules/**;**/logs/**</DefaultSassExcludes>
     
-    <DefaultItemExcludes Condition="'$(EnableDefaultSassItems)'=='true'">$(DefaultItemExcludes);**/*.scss;**/*.sass</DefaultItemExcludes>
+    <!-- 
+      exclude for VS - https://docs.microsoft.com/en-us/visualstudio/msbuild/visual-studio-integration-msbuild#build-solutions 
+      required from this issue (VS bug) - https://github.com/johan-v-r/LibSassBuilder/issues/15
+      -->
+    <DefaultItemExcludes Condition="'$(EnableDefaultSassItems)'=='true' and '$(BuildingInsideVisualStudio)'=='true'">
```

Comment:
@JelleHissink referring to the `BuildingInsideVisualStudio` here - wasn't the only reason for extending this `<DefaultItemExcludes>` **because** of VS (as part of #15) ?
Ideally I wouldn't want this at all. Doubt it actually has any affect on CLI/MSBuild 

Are there scenarios you can think of that we can test?